docs/ref/django-admin.txt

```diff
@@ -106,6 +106,16 @@ For example, to perform only models and compatibility checks, run::
 
     django-admin check --tag models --tag compatibility
 
+
+.. django-admin-option:: --database DATABASES
```

Comment:
This suggests you can pass multiple databases to one `--database` option, but it should be specified multiple times, right?

Reply:
`DATABASES` -> `DATABASE`